shadowsocks-csharp/View/MenuViewController.cs

```diff
@@ -230,6 +285,32 @@ private Bitmap getTrayIconByState(Bitmap originIcon, bool enabled, bool global)
             return iconCopy;
         }
 
+        public int getWindows10SystemThemeSetting()

```

Comment:
Suggested to move to Util.cs with enum return type

Reply:
Already achieved. :)